lzy/execution-env/src/main/java/ai/lzy/env/base/DockerEnvDescription.java

```diff
@@ -18,7 +22,9 @@ public record DockerEnvDescription(
     List<String> envVars,  // In format <NAME>=<value>
     @Nullable
     String networkMode,
-    DockerClientConfig dockerClientConfig
+    DockerClientConfig dockerClientConfig,
+    @Nonnull
```

Comment:
we don't use any notnull annotations
we considr all fields not null by default, else we mark them as `@Nullable`